doc/classes/String.xml

```diff
@@ -209,35 +233,35 @@
 			<argument index="0" name="text" type="String">
 			</argument>
 			<description>
-				Return true if the strings begins with the given string.
+				Returns [code]true[/code] if the string begins with the given string.
 			</description>
 		</method>
 		<method name="bigrams">
 			<return type="PoolStringArray">
 			</return>
 			<description>
-				Return the bigrams (pairs of consecutive letters) of this string.
+				Returns the bigrams (pairs of consecutive letters) of this string.
 			</description>
 		</method>
 		<method name="c_escape">
 			<return type="String">
 			</return>
 			<description>
-				Return a copy of the string with special characters escaped using the C language standard.
+				Returns a copy of the string with special characters escaped using the C language standard.
 			</description>
 		</method>
 		<method name="c_unescape">
 			<return type="String">
 			</return>
 			<description>
-				Return a copy of the string with escaped characters replaced by their meanings according to the C language standard.
+				Returns a copy of the string with escaped characters replaced by their meanings according to the C language standard.
 			</description>
 		</method>
 		<method name="capitalize">
 			<return type="String">
 			</return>
 			<description>
-				Change the case of some letters. Replace underscores with spaces, convert all letters to lowercase then capitalize first and every letter following the space character. For [code]capitalize camelCase mixed_with_underscores[/code] it will return [code]Capitalize Camelcase Mixed With Underscores[/code].
+				Changes the case of some letters. Replaces underscores with spaces, converts all letters to lowercase then capitalizes first and every letter following the space character. For [code]capitalize camelCase mixed_with_underscores[/code] it will return [code]Capitalize Camelcase Mixed With Underscores[/code].
```

Comment:
- lowercase, then
- capitalizes the string's first letter and each letter following a space